jwst/clean_flicker_noise/lib.py

```diff
@@ -399,26 +431,33 @@ def __init__(self, data, mask, fc=(1061, 1211, 49943, 49957),
         # The mask potentially contains statistical outliers.
         # Optionally exclude them.
         if exclude_outliers is True:
-            m, s = med_abs_deviation(self.data[self.mask])  # Compute median and median absolute deviation
+            m, s = med_abs_deviation(
+                self.data[self.mask]
+            )  # Compute median and median absolute deviation
             s *= 1.4826  # Convert MAD to std
-            vmin = m - self.sigrej*s  # Minimum value to keep
-            vmax = m + self.sigrej*s  # Maximum value to keep
+            vmin = m - self.sigrej * s  # Minimum value to keep
+            vmax = m + self.sigrej * s  # Maximum value to keep
 
             # Temporarily change NaNs to inf, so that they don't cause problems.
             self.data[np.isnan(self.data)] = np.inf
 
             # Flag statistical outliers
-            bdpx = np.array(np.where(np.logical_or(self.data < vmin, self.data > vmax),
-                                     1, 0), dtype=np.float32)
+            bdpx = np.array(
+                np.where(np.logical_or(self.data < vmin, self.data > vmax), 1, 0), dtype=np.float32
+            )
             self.data[np.isinf(self.data)] = np.nan  # Restore NaNs
 
-            bdpx[np.logical_not(self.mask)] = 0  # We don't need to worry about non-background pixels
+            bdpx[np.logical_not(self.mask)] = (
+                0  # We don't need to worry about non-background pixels
```

Comment:
placement of comment looks strange to me here, and so does having the 0 on its own line with parentheses, for such a short line